app/policies/podcast_ownership_policy.rb

```diff
@@ -0,0 +1,27 @@
+class PodcastOwnershipPolicy < ApplicationPolicy
+  def update?
+    user_is_owner? || user_admin?
+  end
+
+  def new?
+    true
+  end
+
+  def create?
+    !user_is_banned?
+  end
+
+  def destroy?
+    update?
+  end
+
+  private
+
+  def user_is_owner?
+    if record.instance_of?(PodcastOwnership)
```

Comment:
what's the scenario for it to be something other than a `PodcastOwnership` ?

Reply:
The user who tries to edit the podcast need not be the owner. So if it's not an instance of PodcastOwnership (i.e the user is not the owner of the podcast), they will not be permitted to modify it.

Reply:
I'm not sure I understand fully that it corresponds to the logic you wrote but once you'll write a test for the policy you can test its feasibility. See https://github.com/forem/forem/tree/master/spec/policies for policy specs